crates/tap-agent/src/agent/sender_accounts_manager.rs

```diff
@@ -629,46 +629,83 @@ mod tests {
         },
     };
 
+    use serde_json::json;
+
+    use std::str::FromStr;
+
+    use tap_aggregator::server::run_server;
+
+    use wiremock::matchers::{body_string_contains, method};
+    use wiremock::{Mock, MockGuard, MockServer, ResponseTemplate};
+
     const DUMMY_URL: &str = "http://localhost:1234";
+    const TRIGGER_VALUE: u128 = 100;
+    const ESCROW_VALUE: u128 = 1000;
+    const RAV_REQUEST_BUFFER: Duration = Duration::from_secs(60);
+    const RAV_REQUEST_TIMEOUT: Duration = Duration::from_secs(30);
+    const ESCROW_POLLING_INTERVAL: Duration = Duration::from_secs(30);
+    const TAP_SENDER_TIMEOUT: Duration = Duration::from_secs(63);
+
+    async fn mock_escrow_subgraph_empty_response() -> (MockServer, MockGuard) {
+        let mock_ecrow_subgraph_server: MockServer = MockServer::start().await;
+        let _mock_ecrow_subgraph = mock_ecrow_subgraph_server
+            .register_as_scoped(
+                Mock::given(method("POST"))
+                    //.and(body_string_contains("TapTransactions"))
+                    .respond_with(ResponseTemplate::new(200).set_body_json(json!({ "data": {
+                            "transactions": [],
+                        }
+                    }))),
+            )
+            .await;
+        (mock_ecrow_subgraph_server, _mock_ecrow_subgraph)
+    }
```

Comment:
I had the same thought about "lazy static" here as well: https://stackoverflow.com/a/59614532/24786256